hadoop-hdfs-project/hadoop-hdfs/src/test/java/org/apache/hadoop/hdfs/server/datanode/TestBlockRecovery1.java

```diff
@@ -336,191 +322,194 @@ public void testFinalizedReplicas () throws IOException {
     if(LOG.isDebugEnabled()) {
       LOG.debug("Running " + GenericTestUtils.getMethodName());
     }
-    ReplicaRecoveryInfo replica1 = new ReplicaRecoveryInfo(BLOCK_ID, 
-        REPLICA_LEN1, GEN_STAMP-1, ReplicaState.FINALIZED);
-    ReplicaRecoveryInfo replica2 = new ReplicaRecoveryInfo(BLOCK_ID, 
-        REPLICA_LEN1, GEN_STAMP-2, ReplicaState.FINALIZED);
+    ReplicaRecoveryInfo replica1 = new ReplicaRecoveryInfo(BLOCK_ID,
+        REPLICA_LEN1, GEN_STAMP - 1, ReplicaState.FINALIZED);
+    ReplicaRecoveryInfo replica2 = new ReplicaRecoveryInfo(BLOCK_ID,
+        REPLICA_LEN1, GEN_STAMP - 2, ReplicaState.FINALIZED);
```

Comment:
Can we chunk out the formatting changes in this file? We can restrict ourselves to only the intended changes, Usually we aren't mixing checkstyle fixes with the other code changes..

Reply:
Indeed I also did not want to mix checkstyle with actual test fix, however only after QA bot showed eol space and eol blank warnings, I updated this (in last 2 commits). Are you fine with it?

Reply:
May be because you changed the file name and it read it as a new file? 
one or two fixes are ok, but in case the PR doesn't induce the checkstyle and there are plenty. we can let it stay as is, and let the one approving spend some time, checking that. :-) 
Anyway lets ignore for now. I expect the build won't have anything which would require you any more commits

Reply:
Oh you are right, maybe QA bot had EOL issues which were not relevant to actual changes because I renamed the file earlier.

> Anyway lets ignore for now. I expect the build won't have anything which would require you any more commits

Sure sounds good, I will take care of this going forward. It's the rename which might let QA bot scan entire file regardless of how minor the actual change is in that file.